sync-icloud.sh

```diff
@@ -1731,8 +1733,16 @@ Notify(){
          -H 'Content-Type: application/json; charset=utf-8' \
          -d "{ \"device_key\": \"${bark_device_key}\", \"title\": \"${notification_title}\", \"body\": \"${bark_text}\", \"category\": \"category\" }")"
       curl_exit_code="$?"
+   elif [ "${notification_type}" = "msmtp" ]; then
+      if [ "${notification_files_preview_count}" ]; then
+	      notification_files_preview_text="$(echo "${notification_files_preview_text}" | tr '\n' ',')"
+         mail_text="$(echo -e "${notification_icon} ${notification_message} Most recent ${notification_files_preview_count} ${notification_files_preview_type} files: ${notification_files_preview_text}")"
+      else
+         mail_text="$(echo -e "${notification_icon} ${notification_message}")"
+      fi
+      printf "Subject: $notification_message\n\n$mail_text" | msmtp --host=$msmtp_host --port=$msmtp_port --user=$msmtp_user --passwordeval="echo -n $msmtp_pass" --from=$msmtp_from --auth=on --tls=$msmtp_tls "$msmtp_args" -- "$msmtp_to"
    fi
-   if [ "${notification_type}" ]; then
+   if [ "${notification_type}" != "msmtp" ]; then
```

Comment:
Bug introduced. This is now True if notification type is blank, resulting in attempted notification, when notifications are not configured.

Reply:
Aah, you are right, sorry about that